### PR TITLE
Label even numbered tests

### PR DESCRIPTION
**Pull request by AdamLaszloKulcsar**
Label even numbered tests for clarity's sake.
Test label: 4-80

### CODE REVIEW

File: test_src/sljitTest.c [hunk 1]
```diff
@@ -345,11 +345,14 @@ static void test4(void)
 	buf[3] = 0;
 
 	sljit_emit_enter(compiler, 0, SLJIT_ARGS2(W, P, W), 3, 2, 0, 0, 0);
+    /* buf[0] */
+    sljit_emit_op2(compiler, SLJIT_SUB, SLJIT_MEM0(), (sljit_sw)&buf[0], SLJIT_IMM, 0, SLJIT_MEM0(), (sljit_sw)&buf[1]);
+    /* buf[2] */
```
Comment (zherczeg) [inline test_src/sljitTest.c:350]: Something is wrong with the indentation. Is it space instead of tabs?

File: test_src/sljitTest.c [hunk 2]
```diff
@@ -431,7 +434,7 @@ static void test5(void)
 
 static void test6(void)
 {
-	/* Test addc, sub, subc. */
+	/* Test addc, subc, sub. */
```
Comment (zherczeg) [inline test_src/sljitTest.c:437]: I would keep the sub.subc order

File: test_src/sljitTest.c [hunk 3]
```diff
@@ -455,32 +458,44 @@ static void test6(void)
...
+    /* buf[8] */
 	sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_MEM1(SLJIT_S0), sizeof(sljit_sw) * 8, SLJIT_R1, 0);
-	sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_R0, 0, SLJIT_IMM, 0x52cd3bf4);
-	sljit_emit_op2(compiler, SLJIT_SUB, SLJIT_MEM1(SLJIT_S0), sizeof(sljit_sw) * 9, SLJIT_R0, 0, SLJIT_IMM, 0x3da297c6);
-	sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_R1, 0, SLJIT_IMM, 6000);
+    sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_R0, 0, SLJIT_IMM, 0x52cd3bf4);
+	/* buf[9] */
+    sljit_emit_op2(compiler, SLJIT_SUB, SLJIT_MEM1(SLJIT_S0), sizeof(sljit_sw) * 9, SLJIT_R0, 0, SLJIT_IMM, 0x3da297c6);
+    sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_R1, 0, SLJIT_IMM, 6000);
+	/* buf[10] */
```
Comment (zherczeg) [inline test_src/sljitTest.c:498]: Please fix all indentation issues.

File: test_src/sljitTest.c [hunk 4]
```diff
@@ -2196,22 +2279,29 @@ static void test26(void)
...
 	sljit_emit_op2(compiler, SLJIT_MUL, SLJIT_MEM1(SLJIT_R0), (sljit_sw)sizeof(sljit_sw) * 2 - 103, SLJIT_MEM1(SLJIT_S0), sizeof(sljit_sw) * 2 - 3, SLJIT_MEM1(SLJIT_S0), sizeof(sljit_sw) * 3 - 3);
 	sljit_emit_op2(compiler, SLJIT_ADD, SLJIT_R0, 0, SLJIT_S1, 0, SLJIT_IMM, 100);
+    /* ibuf[2] */
 	sljit_emit_op2(compiler, SLJIT_MUL32, SLJIT_MEM1(SLJIT_R0), (sljit_sw)sizeof(sljit_s32) * 2 - 101, SLJIT_MEM1(SLJIT_S1), sizeof(sljit_s32) * 2 - 1, SLJIT_MEM1(SLJIT_S1), sizeof(sljit_s32) * 3 - 1);
 
 	if (sljit_has_cpu_feature(SLJIT_HAS_FPU)) {
 		sljit_emit_op2(compiler, SLJIT_ADD, SLJIT_S2, 0, SLJIT_S2, 0, SLJIT_IMM, 3);
+        /* dbuf[1] */
 		sljit_emit_fop1(compiler, SLJIT_MOV_F64, SLJIT_MEM1(SLJIT_S2), sizeof(sljit_f64) - 3, SLJIT_MEM1(SLJIT_S2), -3);
+        /* dbuf[2] */
```
Comment (zherczeg) [inline test_src/sljitTest.c:2299]: This also looks as an indentation problem.

File: test_src/sljitTest.c [hunk 5]
```diff
@@ -7387,8 +7600,6 @@ static void test72(void)
 		return;
 	}
 
-	/* Next test. */
```
Comment (zherczeg) [inline test_src/sljitTest.c:7390]: Why are these removed?

File: test_src/sljitTest.c [hunk 6]
```diff
@@ -9570,4 +9838,4 @@ int sljit_test(int argc, char* argv[])
 
 #ifdef _MSC_VER
 #pragma warning(pop)
-#endif
+#endif
```
Comment (zherczeg) [inline test_src/sljitTest.c:9841]: Don't delete the newline at the end.

**Review by zherczeg — CHANGES_REQUESTED** [review]
Nice patch

File: test_src/sljitTest.c (hunk 5 above)
Comment (AdamLaszloKulcsar) [reply test_src/sljitTest.c:7390]: At some places they seemed unnecessary. Should I undo these?

File: test_src/sljitTest.c (hunk 1 above)
Comment (AdamLaszloKulcsar) [reply test_src/sljitTest.c:350]: It was a VS Code indentation problem. It should be fixed now.

File: test_src/sljitTest.c (hunk 5 above)
Comment (zherczeg) [reply test_src/sljitTest.c:7390]: No, you are right. I haven't checked the full context.

File: test_src/sljitTest.c [hunk 7]
```diff
@@ -9549,7 +9818,6 @@ int sljit_test(int argc, char* argv[])
 	test79();
 	test80();
 	test81();
-
```
Comment (zherczeg) [inline test_src/sljitTest.c:9552]: Please keep this newline.

**Review by zherczeg — CHANGES_REQUESTED** [review]
Much better! Please squash commits.

File: test_src/sljitTest.c (hunk 7 above)
Comment (zherczeg) [reply test_src/sljitTest.c:9552]: Please revert this change

**Review by zherczeg — CHANGES_REQUESTED** [review]
Getting close to land. Please resolve the conflicts first.

File: test_src/sljitTest.c (hunk 7 above)
Comment (AdamLaszloKulcsar) [reply test_src/sljitTest.c:9552]: I'm sorry. I forgot about that one. Should be there now.

**Comment by AdamLaszloKulcsar** [comment]
Conflicts should be resolved now and requested changes are in as I believe.

**Review by zherczeg — APPROVED** [review]
LGTM
